api/src/main/java/org/pmiops/workbench/cohortreview/CohortReviewServiceImpl.java

```diff
@@ -128,4 +135,10 @@ public ParticipantCohortStatus findParticipantCohortStatus(Long cohortReviewId,
     public Slice<ParticipantCohortStatus> findParticipantCohortStatuses(Long cohortReviewId, PageRequest pageRequest) {
         return participantCohortStatusDao.findByParticipantKey_CohortReviewId(cohortReviewId, pageRequest);
     }
+
+    @Override
+    public Map<Long, String> findGenderRaceEthnicityFromConcept() {
+        List<Concept> conceptList = conceptDao.findGenderRaceEthnicityFromConcept();
+        return conceptList.stream().collect(Collectors.toMap(Concept::getConceptId, Concept::getConceptName));
```

Comment:
This may not really matter at this scale, but it's kind of annoying to have to reconstruct this map every time the cache gets used... is it possible to tell Hibernate to store this as a Map instead?

FWIW, we use this sort of caching (non-Hibernate) for config stuff today:

https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/config/CacheSpringConfiguration.java#L31

(I have no objections to using Hibernate caching... though it might be better if we eventually went one way or the other for all DB caching.)

Reply:
Ok, so I'm fine with the Guava's cache stuff. Will change to and remove hibernate ehcache